src/config.yml

```diff
@@ -273,6 +275,7 @@ clients:
                 edo: Equations différentielles ordinaires
                 elec: Physique
                 embedded: Software and operating systems for embedded systems
+                embeddedsyst: Systèmes informatiques, aspect temps-réel
```

Comment:
Le nom actuel du cours c'est `Design of embedded and real-time systems`.
Et le nom actuel du cours LINGI2146 (qui a l'air d'être le seul avec la clé "embedded") est `Mobile and embedded computing`.

Reply:
C'est pour matcher avec les noms du drive, je veux bien mettre les noms en anglais, mais je vais changer dans le drive alors.